bin/node-template/node/src/command.rs

```diff
@@ -98,6 +98,15 @@ pub fn run() -> sc_cli::Result<()> {
 				Ok((cmd.run(client, backend), task_manager))
 			})
 		},
+		Some(Subcommand::BenchmarkBlock(cmd)) => {
+			// TODO feature check
```

Comment:
I don't think this strictly needs a feature check right?

We have a feature flag on `runtime-benchmarks` because it actually will add more code to the Wasm runtime, and thus we want to make sure not to have that code bloat the size of the Wasm, but in the case of a purely client side benchmarking tool like this, I think there is no reason not to just include it. 🤷

Reply:
Okay I was not sure how this works exactly, so I put that TODO there.  
Then I will remove it.